x-pack/plugins/cases/server/client/configure/client.ts

```diff
@@ -195,7 +191,7 @@ async function get(
 
         return {
           ...caseConfigureWithoutConnector,
-          connector: transformESConnectorToCaseConnector(connector),
```

Comment:
We no longer need to do the transformation here because that functionality/responsibility has been moved to the service layer.